src/carousel/carousel.component.html

```diff
@@ -1,13 +1,15 @@
-<div (mouseenter)="pause()" (mouseleave)="play()" (mouseup)="play()" class="carousel slide">
+<div (mouseenter)="pause()" (mouseleave)="play()" (mouseup)="play()" class="carousel slide" (focus)="pauseOnFocus()" tabindex="0">
   <ol class="carousel-indicators" *ngIf="showIndicators && slides.length > 1">
     <li *ngFor="let slidez of slides; let i = index;" [class.active]="slidez.active === true" (click)="selectSlide(i)"></li>
   </ol>
   <div class="carousel-inner"><ng-content></ng-content></div>
-  <a class="left carousel-control carousel-control-prev" [class.disabled]="activeSlide === 0 && noWrap" (click)="previousSlide()" *ngIf="slides.length > 1">
+  <a class="left carousel-control carousel-control-prev" [class.disabled]="activeSlide === 0 && noWrap" (click)="previousSlide()" (keydown)="prevKeydown($event)" *ngIf="slides.length > 1"
```

Comment:
are these arrows should listen to `keydown` event? (except for enter) I think that main container should listen to `keydown`, not the arrows, they should only react to `enter`